pkg/sensors/tracing/args_linux.go

```diff
@@ -276,6 +276,19 @@ func getArg(r *bytes.Reader, a argPrinter) api.MsgGenericKprobeArg {
 		arg.SinAddr = network.GetIP(address.SinAddr, address.SinFamily).String()
 		arg.SinPort = uint32(address.SinPort)
 		return arg
+	case gt.GenericSockaddrUnType:
+		var sockaddr api.MsgGenericKprobeSockaddrUn
+		var arg api.MsgGenericKprobeArgSockaddrUn
+
+		err := binary.Read(r, binary.LittleEndian, &sockaddr)
+		if err != nil {
+			logger.GetLogger().Warn("sockaddrun type err", logfields.Error, err)
+		}
+
+		arg.Index = uint64(a.index)
+		arg.Family = sockaddr.Family
+		arg.Path = string(NormalizeSockaddrUnPath(sockaddr.Path[:]))
```

Comment:
It is probably worth noting that this only supports named unix sockets, and we'd need a bit more code to handle abstract sockets, and to separate them from unnamed sockets. That can be a follow-on patch though.

Reply:
My bad; didn't read on!